ui/app/src/state/actions/system.ts

```diff
@@ -112,6 +112,8 @@ export const storeInitialized = /*#__PURE__*/ createAction<{
   user: User;
   sites: Array<Site>;
   properties: LookupTable<any>;
+  activeSiteId?: string;
+  activeEnvironment?: string;
```

Comment:
I don't think anything here is optional. The store initialization is done in a single place and always collects everything.